src/editor/markdown/renderer.js

```diff
@@ -21,6 +21,10 @@ const getRenderer = (context) => {
 const createRenderer = (context) => {
     const markdownItOptions = context && context.options.markdownIt || {html: false, breaks: true, linkify: true};
     const renderer = markdownit(markdownItOptions);
+    renderer.validateLink = function (url) {
+        const BAD_PROTOCOL_REGEXP = /^(vbscript|javascript|data):/;
```

Comment:
@yurabakhtin I would prefer to explicitly whitelist individual protocols instead of blacklisting individual ones. Would that be possible?

Ideally there would be a configuration point where `linkUtil` and `renderer` can access the allowed protocols?